style.css

```diff
@@ -117,6 +117,50 @@ section {
   font-size: 1.2rem;
 }
 
+.sponsor__list {
+  display: inline-flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+.index-sponsors .sponsor__list {
+  display: grid;
+  grid-template-columns: repeat(2, 50%);
+}
+.sponsor__list li {
+  align-self: center;
+  margin-right: 5%;
+}
+
+@media screen and (max-width: 801px) {
+  .index-sponsors .sponsor__list {
+    display: grid;
+    grid-template-columns: repeat(1, 100%);
+  }
+}
+.sponsor__logo {
+  display: inline-block;
+  margin: 2rem 0;
+}
+
+.sponsor__logo__gold img {
+  width: 450px;
+  max-width: 90%;
+}
+
+.sponsor__logo__silver img {
+  width: 375px;
```

Comment:
issue: this produces a horizontal scrollbar on iPhone SE due to limited screen size. Make sure to always have a `max-width` in place (as in gold sponsors)